src/features/welcome/welcome-page.module.scss

```diff
@@ -0,0 +1,166 @@
+.welcome {
+
+    body {
+        margin: 0;
+    }
+
+    :global {
+
+        $title_color: #1417CC;
+        $font-tertiary: #1C284E;
+        $background_light_gray: #F5F7F9;
+        $background_light_green: #C3EEEB;
```

Comment:
From now on you can check if the colors are available in [here](https://github.com/mercury-squad/billy-frontend/blob/dev/src/styles/_variables.scss) and when not, you might add the color in there in case it is used in other parts of the app.